nuxeo/nuxeo.py

```diff
@@ -21,6 +23,10 @@
 
 __all__ = ('Nuxeo',)
 
+logging.basicConfig(format='%(asctime)s %(message)s')
```

Comment:
You can remove this line, it is not the role of the module to set the formatting.